packages/next/src/utils.ts

```diff
@@ -39,9 +39,12 @@ import { getNextjsEdgeFunctionSource } from './edge-function-source/get-edge-fun
 import type { LambdaOptionsWithFiles } from '@vercel/build-utils/dist/lambda';
 import { stringifySourceMap } from './sourcemapped';
 import type { RawSourceMap } from 'source-map';
+import bytes from 'bytes';
 
 type stringMap = { [key: string]: string };
 
+export const prettyKibibytes = (n: number) => bytes(n, { unitSeparator: ' ' });
```

Comment:
**suggestion(non blocking):** can we name this `prettyByte` so it is consistent with this [PR](https://github.com/vercel/edge-functions/pull/419/files#diff-71a2dfec0f0613059ce4d092de5620a4982dc9d4c9edeee3358d981a74836138R9) on `edge-functions`?